pymove/core/grid.py

```diff
@@ -1,26 +1,29 @@
 import math
-import pickle
-import numpy as np
-import pandas as pd
+
+import joblib
 import matplotlib.pyplot as plt
+import numpy as np
 from shapely.geometry import Polygon
 from tqdm import tqdm
-from pymove.utils.conversions import lat_meters
+
 from pymove.utils.constants import (
+    DATETIME,
+    INDEX_GRID_LAT,
+    INDEX_GRID_LON,
     LATITUDE,
     LONGITUDE,
-    DATETIME,
-    TRAJ_ID,
+    POLYGON,
     TID,
-    INDEX_GRID_LON,
-    INDEX_GRID_LAT,
-    POLYGON
+    TRAJ_ID,
 )
+from pymove.utils.conversions import lat_meters
 from pymove.utils.mem import begin_operation, end_operation
 
 
-class Grid():
-    def __init__(self, data, cell_size, meters_by_degree = lat_meters(-3.8162973555)):
+class Grid:
+    def __init__(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__ (D107)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=5452500)